src/main/java/core/basesyntax/suppliers/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.suppliers;
+
+public enum Colors {

```

Comment:
The name of enum must be singular
```suggestion
public enum Color {
```